documentation/modules/exploit/multi/http/atlassian_confluence_rce_cve_2024_21683.md

```diff
@@ -0,0 +1,133 @@
+## Vulnerable Application
+This module exploits an authenticated administrator-level vulnerability in Atlassian Confluence,
+tracked as CVE-2024-21683. The vulnerability exists due to the Rhino script engine parser evaluating
+tainted data from uploaded text files. This facilitates arbitrary code execution. This exploit will
+authenticate, validate user privileges, extract the underlying host OS information, then trigger
+remote code execution. All versions of Confluence prior to 7.17 are affected, as are many versions
+up to 8.9.0.
+
+## Testing
+Download and install a [vulnerable version of Atlassian Confluence](https://www.atlassian.com/software/confluence/download-archives).
+By default, Confluence serves an HTTP service on TCP port 8090. This module was tested against four Confluence installs:
+Linux and Windows Confluence hosts running two different versions, 8.9.0 and 7.20.2. The target host operating systems
+were Ubuntu 22.04 and Server 2022.
+
+## Verification Steps
+Note: Disable Defender if you are using the default payloads.
+
+Steps:
+1. Start msfconsole
+2. `use exploit/multi/http/atlassian_confluence_rce_cve_2024_21683`
+3. `set RHOST 192.168.156.131`
+4. `check`
+5. `set LHOST 192.168.156.129`
+6. `set ADMIN_USER admin`
+7. `set ADMIN_PASS Password123!`
+8. For Windows targets, `set FETCH_COMMAND CERTUTIL` is recommended. For Linux targets, `set FETCH_COMMAND CURL` is recommended.
+9. `exploit`
+
+## Options
+
+### ADMIN_USER
+
+The known Confluence administrator username.
+
+### ADMIN_PASS
+
+The known Confluence administrator password.
+
+## Scenarios
+
+### Windows Target
```

Comment:
Very minor but could you please specify the version here as well as the Flavour and version of Linux below, ex: 
`### Windows 10 (Build 15063)`


Also in the scenarios section once a session is obtained we usually try to include the output of `getsysinfo` (if it's a meterpreter session) or the first part of `systeminfo` or `uname -a` as it helps future travellers reproduce the environment the module was tested on.

Reply:
Sure thing!